src/plugins/initiatives/config.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import * as C from "../../util/combo";
+import {_validateUrl} from "./initiativesDirectory";
+
+export type InitiativesConfig = {|
+  +remoteUrl: string,
+|};
+
+export function parseConfig(raw: C.JsonObject): InitiativesConfig {
+  return configParser.parseOrThrow(raw);
+}
+
+export const configParser: C.Parser<InitiativesConfig> = C.object({
+  remoteUrl: C.fmap(C.string, _validateUrl),
+});
```

Comment:
I usually just export the parsers as `parser`, and then import with the needed name.